README.md

```diff
@@ -182,7 +150,7 @@ null;
 
 undefined;
 ```
-  
+
 ## Return Values
```

Comment:
@ipc103 sounds good to me - there is a functions codealong/readme, so let's just make sure its explicitly stated in that readme!